client-gateway/bot/passenger.py

```
@@ -370,67 +231,224 @@ def handle_order_status(call):
                     code = (error or {}).get('status_code')
-<<<<<<< HEAD
+                    local_req = req_id if req_id is not None else 'N/A'
                     await query.edit_message_text(
-=======
-                    bot.edit_message_text(
-                        chat_id=chat_id,
-                        message_id=call.message.message_id,
->>>>>>> main
                         text=(
-                            "\u274C **Не вдалося створити поїздку.**\n"
+                            "❌ Не вдалося створити поїздку.\n"
```

Ruff (0.14.10)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)